### PR TITLE
Hitting enter no longer sends a chat message, instead inserts a new line

### LINKED ISSUE #8066: Enter/Return key should not post chat messages
Currently, when typing a message in the Tavern, guilds, or party, hitting the keyboard's Enter key will post the message. This causes users who aren't familiar with this behaviour to post multiple messages when they meant to post only one.

**This should be changed so that Enter does nothing but make a new line (as Shift-Enter does now).**

This is important because we want to introduce an anti-spam limit on posts (https://github.com/HabitRPG/habitrpg/issues/8060) and we don't want users to accidentally exceed the limit by posting messages when they don't mean to.

Users who want to use keyboard controls to post messages can do Tab-Enter or Tab-Space (Tab moves focus to the send button and Enter or Space "clicks" the button). This behaviour must keep working. We can inform the Socialites about that so that they can share it to people who aren't happy with the removal of the Enter key's function.

(We could change Shift-Enter so that it posts the message, but many users are already used to using Shift-Enter to insert line breaks, so I personally think that changing that isn't a good idea at the moment. Perhaps a few months after Enter stops posting messages we could consider making Shift-Enter post them.)

Any coder who wants to work on this is welcome to post here to claim it, but please do that only if you can start now and finish ideally in a couple of weeks. Please give a progress report after a week so your claim doesn't expire.

FYI @lemoness

### PR DESCRIPTION
**Pull request by Hus274**
Fixes #8066 
### Changes

changing behavior so hitting enter in a chat box only now inserts a newline instead of submitting the form.

The user can now tab to send chat and click enter or use Control(command) + enter to submit thanks to Blade's feature here: https://github.com/HabitRPG/habitica/issues/4460

### CODE REVIEW

**Comment by crookedneighbor** [comment]
The send chat button should have a tooltip that describes how to send a chat from the keyboard. We should also be sure that we can tab to the send button from the text area.

**Comment by Hus274** [comment]
I tested the ability to tab to the send button from the text area and it worked for me. I'll add in the tool tip for the submit button here.

**Comment by Hus274** [comment]
Added the tool tip message. It's a bit wordy and might need a bit of word smithing. Just let me know and I can update it!

I was also thinking that it could potentially be a loading screen tip!

**Comment by crookedneighbor** [comment]
Tier awarded! You can read more about [contributor tiers and rewards here](http://habitrpg.wikia.com/wiki/Contributor_Rewards).

Note that as your tier increases, it takes progressively more work to achieve the next one, so for the higher tiers, larger PRs or a few PRs are required. There's some information about that at Contributor Rewards, but feel free to ask here if you have questions!
